src/keygen/mod.rs

```diff
@@ -10,3 +10,6 @@
 pub(crate) mod keygen_commit;
 pub(crate) mod keyshare;
 pub mod participant;
+
+pub use keyshare::{KeySharePrivate, KeySharePublic};
+pub use participant::Output;
```

Comment:
note: Public and private key shares are not actually used directly by any of our examples right now, but I anticipate that when we start doing secure storage, we will need to break down the `Output` type into its constituent public and private parts.

Reply:
See #376.